cli/ssh.go

```diff
@@ -72,6 +73,11 @@ func ssh() *cobra.Command {
 				return err
 			}
 
+			updateWorkspaceBanner, outdated := verifyWorkspaceOutdated(client, workspace)
+			if outdated {
+				_, _ = fmt.Fprintln(cmd.OutOrStdout(), updateWorkspaceBanner)
```

Comment:
We should change this to print on stderr so that non-interactive commands like `ssh coder.workspace ls | grep hi` aren't dirtied.

I wonder if we should consider only outputting when the recipient is a tty as well. 🤔

Reply:
Good catch!

Switched to stderr and added a TTY check.